Pixel Game/Assets/Scripts/DontDestroyTestScripts/CMvcam1DontDestroy.cs

```diff
@@ -1,12 +1,24 @@
-﻿using System.Collections;
+﻿using Cinemachine;
+using System.Collections;
 using System.Collections.Generic;
 using UnityEngine;
+using UnityEngine.SceneManagement;
 
 public class CMvcam1DontDestroy : MonoBehaviour
 {
     private static bool cMvcam1Exists;
+    private GameObject player;
+    private CinemachineVirtualCamera cinemachineVirtualCamera;
+    private string nameOfMainMenu = "Main_Menu_Lasse";
     void Start()
     {
+        if(SceneManager.GetActiveScene().name != nameOfMainMenu)
+        {
+            player = GameObject.FindGameObjectWithTag("MyPlayer");
+            cinemachineVirtualCamera = GameObject.FindGameObjectWithTag("VCam").GetComponent<CinemachineVirtualCamera>();
+            cinemachineVirtualCamera.Follow = player.transform;

```

Comment:
Since this is the only place where `player` is used, it could be a local variable.

Reply:
I dont really get what you mean here?...

Reply:
Instead of declaring it at the top of the class as `GameObject player` it could be declared right here as `var player = GameObject.FindGameObjectWithTag("MyPlayer");`

Reply:
And the difference between var and gameobject will be..??

Reply:
No difference, except var is shorter. But the main point is that it could be declared locally. There is no point declaring it outside of the scope where it is used.

Reply:
Aaaah now i get what you are saying! I missed the top of your sentence :D